platform/default/include/mbgl/storage/offline_schema.hpp

```diff
@@ -35,6 +35,35 @@ static constexpr const char* offlineDatabaseSchema =
     "  must_revalidate INTEGER NOT NULL DEFAULT 0,\n"
     "  UNIQUE (url_template, pixel_ratio, z, x, y)\n"
     ");\n"
```

Comment:
This file is generated and should not be modified manually.

Reply:
Could you try updating `offline_schema.js` and running that?

Reply:
OK, I didn't know that the file was autogenerated, sorry, I will try to do that :-)